code/__DEFINES/food.dm

```diff
@@ -1,18 +1,23 @@
-#define MEAT 		(1<<0)
-#define VEGETABLES 	(1<<1)
-#define RAW 		(1<<2)
-#define JUNKFOOD 	(1<<3)
-#define GRAIN 		(1<<4)
-#define FRUIT 		(1<<5)
-#define DAIRY 		(1<<6)
-#define FRIED 		(1<<7)
-#define ALCOHOL 	(1<<8)
-#define SUGAR 		(1<<9)
-#define GROSS 		(1<<10)
-#define TOXIC 		(1<<11)
-#define PINEAPPLE	(1<<12)
-#define BREAKFAST	(1<<13)
-#define CLOTH 		(1<<14)
+#define MEAT (1<<0)
+#define VEGETABLES (1<<1)
+#define RAW (1<<2)
+#define JUNKFOOD (1<<3)
+#define GRAIN (1<<4)
+#define FRUIT (1<<5)
+#define DAIRY (1<<6)
+#define FRIED (1<<7)
+#define ALCOHOL (1<<8)
+#define SUGAR (1<<9)
+#define GROSS (1<<10)
+#define TOXIC (1<<11)
+#define PINEAPPLE (1<<12)
+#define BREAKFAST (1<<13)
+#define CLOTH (1<<14)
+/*#define NUTS (1<<15)
```

Comment:
we really don't need internal consistency on flags with TG btw, the numbers don't matter since we don't persist objects (Yet)

Reply:
Yeah, I'm getting ahead of myself a bit here. 

I'm planning on adding these foodgroups after Part 4 of newfood, so I decided to keep number consistency and comment them out for now.